tooling/nargo_cli/src/cli/info_cmd.rs

```diff
@@ -121,6 +144,51 @@ pub(crate) fn run(
     Ok(())
 }
 
+fn print_span_opcodes(
```

Comment:
I am not sure of what we are supposed to print.
Could you add some comments about what the function is supposed to output?
It's not clear to me what is the relevance to display the length of the call stack for each opcode.

Reply:
053fa94: chore: add print fn comment